src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -60,7 +60,7 @@ public JwtHeader()
         /// <para>&#160;&#160;&#160;&#160;'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256' => 'RS256'</para>
         /// <para>&#160;&#160;&#160;&#160;'http://www.w3.org/2001/04/xmldsig-more#hmac-sha256' => 'HS256'</para>
         /// </remarks>
-        public JwtHeader(SigningCredentials signingCredentials)
+        public JwtHeader(SigningCredentials signingCredentials, IDictionary<string, string> outboundAlgorithmMap = null)
```

Comment:
I added this optional parameter for instance based maps to actually work. We can revisit this when we tackle the signing credentials problems.